ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountReportsDao.java

```diff
@@ -123,6 +132,61 @@ void queryStorageUsageForHost(String clusterName, String hostname, ContainerUsag
     }
   }
 
+  void deleteStorageUsageForPartition(String clusterName, String hostname, int partitionId) throws SQLException {
```

Comment:
It will not, I am not add these methods to the callback function of partition state transition because we already can do it by just comparing the previous stats and current stats.